Workday/canvas-kit
# fix: Basic Grid Example with landmark regions for screen readers

Enhancing basic grid example by using as prop for semantic landmark regions for screen readers in the layout.



## Summary



I wanted the basic grid example to yield semantic elements for the page layout that can be used as landmark regions for screen readers. 


## Release Category
Documentation

### Release Note
Optional release note message. Changelog and release summaries will contain a pull request title. This section will add additional notes under that title. This section is not a summary, but something extra to point out in release notes. An example might be calling out breaking changes in a labs component or minor visual changes that need visual regression updates. Remove this section if no additional release notes are required.

### BREAKING CHANGES
Optional breaking changes message. If your PR includes breaking changes. It is extremely rare to put breaking changes outside a `prerelease/major` branch. Anything in this section will show up in release notes. Remove this section if no breaking changes are present.

---

## Checklist

- [ X ] MDX documentation adheres to Canvas Kit's [standard MDX template](https://github.com/Workday/canvas-kit/discussions/1131)
- [ X ] Label `ready for review` has been added to PR

## For the Reviewer




## Where Should the Reviewer Start?




## Areas for Feedback? (optional)





## Testing Manually


- Header is a `<header>` element,
- BodyContent is a `<main>` element,
- SideBar is an `<aside>` element,
- Footer is a `<footer>` element
AND
When using a screen reader, you can jump each section of the layout. With VoiceOver on MacOS:
1. Set rotor to "landmark" => Ctrl + Opt + Cmd + ArrowRight / Left
2. Jump to next region => Ctrl + Opt + Cmd + ArrowDown (ArrowUp for previous region)
3. Use web rotor => Ctrl + Opt + U
4. Find "Landmarks" menu => ArrowRight / Left
5. Observe 4 regions of the grid layout

## Screenshots or GIFs (if applicable)



## Thank You Gif (optional)
